### PR TITLE
Improve ping attribute explanation

### LINKED ISSUE #1456: Privacy concern with ping attribute
Originally filed by [email from Nick Doty](https://lists.w3.org/Archives/Public/public-html/2018May/0027.html), in response to a wide review request to the Privacy IG.

>it looks like the ping attribute has re-appeared (from #369 .
>
>When this was discussed in the past, I believe privacy concerns were specifically cited as a reason not to include this in the updated HTML5 spec, but
it seems to have gone ahead on this draft, based on increased implementation experience.
>
>When it was discussed with the Privacy Interest Group in April 2017, a specific comment was noted:
>>"Note that if we have a requirement that user agents clarify to the user that the link will ping other sites, we need to test whether that happens in
real implementations."
>
>That concern stands. 
>
>Here is the relevant text from the spec (present in both WHATWG HTML and HTML 5.3 WD):
>>"user agents should make it clear to the user that following the hyperlink will also cause secondary requests to be sent in the background."
>
>Has anyone tested the real implementations to verify that user agents clarify to the user that the link will also cause secondary requests to be sent in the background? In my quick checks on current versions of Chrome and Safari on Mac OS clicking on links from a google.com search results page, I couldn't determine that secondary requests were being sent in the background short of opening the network inspector and observing HTTP POST requests. I trust that we don't believe that is "clear to the user". (The spec includes an example suggesting use of a tooltip; I'm not sure that's very clear either, but I haven't seen that in existing implementations anyway.) Do others have tests/screenshots/etc. documenting implementations that fulfill this requirement?
>
> Indeed, the question of implementing the clarity requirement was [raised in 2007](https://lists.w3.org/Archives/Public/public-html/2007Nov/0055.html), with the suggestion that if sufficient UI wasn't being implemented within a year, then the feature should be revisited:
>
>If there haven't been compliant implementations developed in the past 10 or 11 years, then I question whether there is sufficient implementation experience or whether implementers believe that feasible UI is possible to meet that requirement.
>
>I believe there are good motivations for the ping attribute feature in a way that could help user privacy. >However, I think we need to be cautious about a feature where the privacy UI hasn't been developed for this long. Indeed, this might be a step backward in transparency for end users, who in some cases now can no longer use the status bar to observe that they're being redirected through a tracking link, and might conclude that tracking isn't happening, that they're navigating to a site by clicking a link without any background communications. In neither browser I tested could I find a setting to disable sending these background requests, as was proposed as an advantage of the feature.
>
>If implementers believe that the clarity requirement is actually unnecessary and that it's still better for user privacy, that would be a separate question to discuss. That approach might better match the reality of implementations, but I'm not sure what the privacy advantage is if users have neither awareness nor control of background tracking requests.

### PR DESCRIPTION
**Pull request by chaals**
fix #1456 

Make the author encouragement a formal **should** (effectively a substantive change but no requirement on browser implementation)

Clarify the potential benefits and the current reality.

### CODE REVIEW

**Comment by scottaohara** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by scottaohara** [comment]
Looks good to me but for the change.include conflict.

File: sections/semantics-links.include [hunk 1]
```diff
@@ -1015,21 +1015,22 @@
...
-  </div>
+    The functions of the <{links/ping}> attribute such as tracking which links are most popular,
+    or allowing advertisers to track click-through rates, can be achieved with HTTP redirects,
+    Javascript analytic software, and the like.
+
+    Nevertheless authors should use the <{links/ping}> attribute for hyperlink auditing,
+    as it provides several advantages, in particular to give users control over their privacy.
+
+    * Because the final target URL is in the link,
+      the user has more certainty about where they will navigate to.
```
Comment (siusin) [inline sections/semantics-links.include]: Bikeshed complains about the line-break here...

File: sections/semantics-links.include [hunk 2]
```diff
@@ -1015,21 +1015,22 @@
...
+    * User agents, or extensions, could reliably inform the user about tracking notifications,
+      and allow the user to disable them on a case-by-case basis.
```
Comment (siusin) [inline sections/semantics-links.include]: Ditto.

Comment (chaals) [reply sections/semantics-links.include]: fixed them.

**Pull request merged**